playbooks/ansible-test-network-integration-eos/files/bootstrap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: controller
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Install pexpect requirement
+      pip:
+        name: pexpect
+        extra_args: --user
+
+- hosts: controller
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Enable eAPI access with default configuration
```

Comment:
Does anybody have a better way of doing this? I couldn't get this to work via eos_eapi because it seems to force an vrf.